### ISSUE TITLE
Why do you use SelectPercentile and f_classif?

### ISSUE THREAD

**Issue by alexxsstar** [CynthiaKoopman/Network-Intrusion-Detection#2]
I can't understand why you use SelectPercentile and f_classif in feature selection? After all, you do not use the results of this selection in classifier training?

**Comment by dMaisnam** [comment]
[**In cell number 23** ]It selects the top 10 percentage (13 features) of correlated features and stores it in the X_newDoS, X_Probe, X_R2L, X_U2R variables which are used in training the model [** In cell number 32**] with 'clf' variable.

**Comment by alexxsstar** [comment]
Okay, i understand it. The next step is feature selection with RFE (Option 2). On the this step we do not use X_newDos and etc. We use full data (X_DoS...) and result selection we test at the last step. But selection with SelectPercentile we don't estimate.

In this way, i don't understand why need RFE - Option 1? Yes, we sorted features which we got with SelectPercentile, but this result don't use in the next steps...

I read the whole original article some times but i can't understand this :(
